include/deal.II/lac/solver.h

```diff
@@ -214,8 +214,8 @@ class Vector;
  * more specifics are discussed in the documentation of the connect()
  * function). A pointer to a function with this argument list satisfies the
  * requirements, but you can also pass a member function whose
- * <code>this</code> argument has been bound using the
- * <code>std::bind</code> mechanism (see the example below). - Each of
+ * <code>this</code> argument has been bound using a lambda function
+ * (see the example below). - Each of
```

Comment:
That's not your fault, but I think we used to have an itemized list here that was at one point destroyed by the indentation script.